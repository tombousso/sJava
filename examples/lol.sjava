(package sjava.examples.lol)

(define-class Main () 'public
	((main args String[]) void 'public 'static
		(define x 3)
		(
			(lambda ()
				(begin
					(println x)
					(println x)
				)
				(
					(lambda ()
						(print x)
						(println ((aget ((Thread:currentThread):getStackTrace) 1):getLineNumber))
						(println x)
					)
				)
			)
		)
	)
)
