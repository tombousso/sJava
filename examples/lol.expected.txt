3
3
315
3
